app/views/layouts/_nav.html.haml

```diff
@@ -1,16 +1,22 @@
 - simplified_nav = false if !defined?(simplified_nav)
-
 - if simplified_nav
   - pages = ['Home']
 - else
   - pages = ['About', 'Gallery', 'Schedule', 'FAQ', 'Sponsors']
 
+- api_list = current_page?(:controller => 'home', :action => 'apilist')
+- if api_list
+  - pages = ['Home']
+
+
 %nav.desktop
   .logo
     %a{ href: '#' }
       = image_tag 'brickhack.png', { class: 'wow rotate-in', alt: 'BrickHack' }
   .nav-links
     - pages.each do |page|
+      - if api_list
+        %a{ href: "/" }= page
       - if simplified_nav
```

Comment:
`- if api_list || simplified_nav` - otherwise it'll duplicate nav items

Edit: this could probably be simplified even more by integrating the logic on line 2 but not super important

```haml
- api_list = current_page?(:controller => 'home', :action => 'apilist')
- if simplified_nav || api_list
  - pages = ['Home']
- else
...
```